README.md

```diff
@@ -154,8 +154,17 @@ Example Playbook
                   format: 'qcow2'
                   capacity: '200GB'
                   pool: 'my-pool'
+                - name: 'filestore'
```

Comment:
Would it be more flexible to just allow the file path to be configured? It's not obvious to me from the example that the volume name gets transformed to become a filename.

Reply:
I've changed the description of 'name' in 2e165d4 to mention it will become part of the filename. I have also made the images folder a default value so it can be changed.
I thought about making the images {{name}}.{{format}} but some sites might not want an extension at all so this way seems safer.